src/main.c

```diff
@@ -84,10 +83,16 @@ int renderer_run(struct renderer* r) {
 		tv.tv_nsec = tv.tv_nsec % 1000000000;
 	}
 
+	LOG(("Turning off LEDs\n"));
+
+	memset(r->framebuf, 1, sizeof(r->framebuf));
```

Comment:
Ya, sending all 0x00 didn't work, so I tried 0x01 and that seemed to work. 

Where can I find specs for the system?